src/pages/patterns/global-header-pattern/index.mdx

```diff
@@ -0,0 +1,297 @@
+---
+title: Global header
+description: Users rely on the global header to navigate and orient themselves in your UI. This pattern outlines some of the qualities that make a global header consistent, familiar, and usable.
+---
+
+### Users rely on the global header to navigate and orient themselves in your UI. This pattern outlines some of the qualities that make a global header consistent, familiar, and usable.
+
+<AnchorLinks>
+
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Configurations</AnchorLink>
+<AnchorLink>Navigation</AnchorLink>
+<AnchorLink>Best Practices</AnchorLink>
+<AnchorLink>Accessibility</AnchorLink>
+<AnchorLink>Related components</AnchorLink>
+<AnchorLink>References</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+### Header and panel regions
+
+The region describes the location and persistence of header elements in your UI.
```

Comment:
global or local in not a region.
it’s about persistance, availability.
a header is a persistent, globally available element. within it there are regions for product, or property tasks (to the left) and for system tasks (on the right).